### DISCUSSION TITLE
Remapping ESC behavior

### DISCUSSION THREAD

**Discussion by simonmandlik** [wezterm/wezterm#5438] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Is it possible to remap "ESC" key to "CTRL-[" so that both have the same behavior? For example, to exit "PaneSelect" it is now possible to press only the Escape key.

I'm migrating from tmux, which makes these two identical

**Comment by kdien** [comment]
Usually for this type of remap, it's better to do on the OS level. For macOS, you could try looking into Karabiner.

**Reply by simonmandlik** [reply]
Thanks for the suggestion, but I like to keep terminal-based apps separate from OS (in case the shortcut is important in another application)
